src/lib/redux-open-api.test.ts

```diff
@@ -96,8 +97,8 @@ describe('http client', () => {
   test('can request', () => {
     return HttpClient({
       url: 'http://localhost/pets',
-    }).then((res) => {
-      assert(res.status, 200);
+    }).then((res: TODO) => {
+      assert(res.status, '200');
```

Comment:
すみません。ここのassertの使い方間違ってました・・・
```suggestion
      assert.strictEqual(res.status, 200);
```